hack/bootstrap-buildx

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+topdir="$(realpath $(dirname "$0")/..)"
+cd "${topdir}"
+
+NAME="moby-buildkit"
+echo "Bootstrapping buildx builder \"${NAME}\""
+
+buildx="./bin/buildx"
+if [ ! -f "${HOME}/.docker/buildx/instances/${NAME}" ]; then
+  set -x
+  "${buildx}" create --name "${NAME}"
+fi
+
+set -x
+"${buildx}" inspect --bootstrap "${NAME}"
+"${buildx}" use "${NAME}"
```

Comment:
maybe `export BUILDX_BUILDER=` better here to not make opinionated changes to user's environment

Reply:
updated